<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>üéß VoiceChat</string>
  </property>
  <property name="styleSheet">
   <string>
    QMainWindow { background-color: #1e1e2e; }
    QLabel { color: #cdd6f4; }
    QLineEdit {
      background-color: #313244;
      border: 2px solid #45475a;
      border-radius: 8px;
      padding: 8px;
      color: #cdd6f4;
      font-size: 13px;
    }
    QLineEdit:focus { border: 2px solid #89b4fa; }
    QPushButton {
      border: none;
      border-radius: 8px;
      padding: 10px;
      font-weight: bold;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainHLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    
    <!-- SOL PANEL - Mikrofon + Kulaklƒ±k -->
    <item>
     <widget class="QFrame" name="leftPanel">
      <property name="styleSheet">
       <string>
        QFrame { 
          background-color: #181825; 
          border-right: 1px solid #313244;
        }
       </string>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>15</number>
       </property>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
       
       <!-- Mikrofon ve Kulaklƒ±k yan yana -->
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="micButton">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>üé§</string>
           </property>
           <property name="styleSheet">
            <string>
             QPushButton {
               background-color: #45475a;
               border-radius: 30px;
               font-size: 28px;
             }
             QPushButton:hover { background-color: #585b70; }
             QPushButton:disabled { background-color: #313244; }
            </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="audioButton">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>üéß</string>
           </property>
           <property name="styleSheet">
            <string>
             QPushButton {
               background-color: #45475a;
               border-radius: 30px;
               font-size: 28px;
             }
             QPushButton:hover { background-color: #585b70; }
             QPushButton:disabled { background-color: #313244; }
            </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       
       <!-- Durum yazƒ±larƒ± -->
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="micStatusLabel">
           <property name="text">
            <string>Mik</string>
           </property>
           <property name="styleSheet">
            <string>font-size: 10px; color: #6c7086;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="audioStatusLabel">
           <property name="text">
            <string>Ses</string>
           </property>
           <property name="styleSheet">
            <string>font-size: 10px; color: #6c7086;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    
    <!-- SAƒû PANEL - Kontroller -->
    <item>
     <widget class="QFrame" name="rightPanel">
      <property name="styleSheet">
       <string>QFrame { background-color: #1e1e2e; }</string>
      </property>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>20</number>
       </property>
       <property name="spacing">
        <number>12</number>
       </property>
       
       <!-- Ba≈ülƒ±k ve Durum -->
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="titleLabel">
           <property name="text">
            <string>Voice Chat</string>
           </property>
           <property name="styleSheet">
            <string>font-size: 18px; font-weight: bold; color: #cdd6f4;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="statusLabel">
           <property name="text">
            <string>‚óè √áevrimdƒ±≈üƒ±</string>
           </property>
           <property name="styleSheet">
            <string>font-size: 11px; color: #f38ba8;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       
       <!-- IP ve Baƒülan yan yana -->
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLineEdit" name="serverIpEdit">
           <property name="placeholderText">
            <string>Server IP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="onlineButton">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>38</height>
            </size>
           </property>
           <property name="text">
            <string>Baƒülan</string>
           </property>
           <property name="styleSheet">
            <string>
             QPushButton {
               background-color: #a6e3a1;
               color: #1e1e2e;
               font-size: 12px;
             }
             QPushButton:hover { background-color: #94e2d5; }
            </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       
       <!-- Bo≈üluk -->
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
       
       <!-- Alt kƒ±sƒ±m - Host -->
       <item>
        <widget class="QFrame" name="hostFrame">
         <property name="styleSheet">
          <string>
           QFrame {
             background-color: #313244;
             border-radius: 10px;
           }
          </string>
         </property>
         <layout class="QHBoxLayout">
          <property name="margin">
           <number>10</number>
          </property>
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="hostTitleLabel">
              <property name="text">
               <string>Lokal Sunucu</string>
              </property>
              <property name="styleSheet">
               <string>font-size: 11px; color: #bac2de; font-weight: bold;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="hostStatusLabel">
              <property name="text">
               <string>Kapalƒ±</string>
              </property>
              <property name="styleSheet">
               <string>font-size: 10px; color: #6c7086;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="hostButton">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Ba≈ülat</string>
            </property>
            <property name="styleSheet">
             <string>
              QPushButton {
                background-color: #45475a;
                color: #cdd6f4;
                font-size: 11px;
                border-radius: 6px;
              }
              QPushButton:hover { background-color: #585b70; }
             </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       
       <!-- Ana Server IP bilgisi -->
       <item>
        <widget class="QLabel" name="infoLabel">
         <property name="text">
          <string>Ana Sunucu: 46.225.8.192:50000</string>
         </property>
         <property name="styleSheet">
          <string>font-size: 10px; color: #585b70;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       
      </layout>
     </widget>
    </item>
    
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
